aw_watcher_window/printAppStatus.jxa

```diff
@@ -42,18 +42,24 @@ switch(appName) {
     title = Application(appName).windows[0].name();
     break;
   default:
-    mainWindow = oProcess.
-      windows().
-      find(w => w.attributes.byName("AXMain").value() === true)
+    try {
+      mainWindow = oProcess.windows().find((w, i) => w && w.attributes && w.attributes() && w.attributes.byName("AXMain") && w.attributes.byName("AXMain").value() === true)
+    } catch {
+        mainWindow = oProcess.windows()[0]
+      };
+    
 
     // in some cases, the primary window of an application may not be found
     // this occurs rarely and seems to be triggered by switching to a different application
```

Comment:
It would be great if you could throw in the explanation you have in the PR description as a code comment (optionally with a link to the PR), so that people who read it in the future can understand the context/reason :)